.eslintignore

```diff
@@ -0,0 +1,6 @@
+# don't ever lint node_modules
```

Comment:
Why you reintroduce eslint since we have it in https://github.com/jitsi/jibri-queue/pull/5/files ?

Reply:
This is config specific to typescript and is branched of master which doesn't have eslint yet. They will eventually get merged. We probably won't be merging the linked pull request into master as is and will most likely pull from it what we need in subsequent PRs.